content/docs/ref/index.md

```diff
@@ -1,4 +1,17 @@
-# Command Reference
+# CML Command Reference
```

Comment:
why the unnecessary "CML" prefix? It's already in the domain name and page title.

Reply:
No strong opinion but we should try to be consistent? Currently:

- /doc = **CML Documentation**
- /doc/start/github = ** Get Started with CML on GitHub**
- /usage = **Using CML**
- etc.

Reply:
very true, I think we should remove it from everywhere (because ` | CML · Continuous Machine Learning` is already auto-suffixed to all pages):

- /doc = Documentation
- /doc/start/github = Get Started on GitHub
- /usage = Usage
- etc.

Reply:
```suggestion
# Command Reference
```

OK. Rolling back here ☝🏼 and extracted task to https://github.com/iterative/cml.dev/issues/401.